api/v1alpha1/edgedeployment_webhook_test.go

```diff
@@ -0,0 +1,174 @@
+package v1alpha1_test
+
+import (
+	"github.com/jakub-dzon/k4e-operator/api/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+var _ = Describe("v1alpha1", func() {
```

Comment:
I would change this name to edgeDeployment Webhook, so it's clear when something fails.

Reply:
r u refering to edgeDeployment variable?

Reply:
got it. changing